### ISSUE TITLE
Maps

### ISSUE THREAD

**Issue by patrickkenney9801** [firejake308/code-versus-bugs#68]
We need at least one more map to add diversity and make the game easier for new players.

**Comment by firejake308** [comment]
I disagree, since the minigame will add this diversity while also consuming
most of our limited resources.

**Comment by firejake308** [comment]
If anything, I think it would be wiser to diversify the map that we already
have by providing each entrance with a "personality," or a tendency to rely
on a certain type of malware.
